### PR TITLE
Kp pytools to warptools pd 2186

### PR DESCRIPTION
**Pull request by kevinpalis**
We have already moved the custom python scripts and the docker file for the pytools image into warp-tools. We do not want to have duplicate code.

This PR will remove all usage of pytools in WARP and replace them with the latest warp-tools image that is semantically versioned (ie. not a test container). This also changes the path of the scripts in the container so a minor change to command calls is necessary. 
----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

File: pipelines/skylab/scATAC/scATAC.wdl [hunk 1]
```diff
@@ -291,7 +291,7 @@ task MakeCompliantBAM {
 task BreakoutSnap {
     input {
         File snap_input
-        String docker_image = "us.gcr.io/broad-gotc-prod/pytools:1.0.0-1661263730"
+        String docker_image = "us.gcr.io/broad-gotc-prod/warp-tools:1.0.1-1686932671"
```
Comment (kayleemathews) [inline pipelines/skylab/scATAC/scATAC.wdl:294]: Can you update the link to the python script for this task here?: https://github.com/broadinstitute/warp/blob/develop/website/docs/Pipelines/Single_Cell_ATAC_Seq_Pipeline/README.md#makecompliantbam

File: tasks/skylab/LoomUtils.wdl [hunk 2]
```diff
@@ -169,7 +169,7 @@ task AggregateSmartSeq2Loom {
         String? species
         String? organ
         String pipeline_version
-        String docker = "us.gcr.io/broad-gotc-prod/pytools:1.0.0-1661263730"
+        String docker = "us.gcr.io/broad-gotc-prod/warp-tools:1.0.1-1686932671"
```
Comment (kayleemathews) [inline tasks/skylab/LoomUtils.wdl:172]: Can you update the link to the script used in this task in the [multi-snSS2 Task table](https://github.com/broadinstitute/warp/blob/develop/website/docs/Pipelines/Smart-seq2_Single_Nucleus_Multi_Sample_Pipeline/README.md#multi-snss2-tasks-and-tools) (I think it's mentioned twice) and in the [outputs section](https://github.com/broadinstitute/warp/blob/develop/website/docs/Pipelines/Smart-seq2_Single_Nucleus_Multi_Sample_Pipeline/README.md#7-outputs) (1 mention)?

**Review by kayleemathews — COMMENTED** [review]
Hey @kevinpalis, would you mind making a couple of doc updates to go along with these changes? If you have any questions please let me know. Thanks so much!

File: pipelines/skylab/scATAC/scATAC.wdl (hunk 1 above)
Comment (kevinpalis) [reply pipelines/skylab/scATAC/scATAC.wdl:294]: Sure thing! Thanks Kaylee!

File: tasks/skylab/LoomUtils.wdl (hunk 2 above)
Comment (kevinpalis) [reply tasks/skylab/LoomUtils.wdl:172]: Will do

File: pipelines/skylab/scATAC/scATAC.wdl (hunk 1 above)
Comment (kevinpalis) [reply pipelines/skylab/scATAC/scATAC.wdl:294]: Hi Kaylee. I made the changes. Thanks!

File: tasks/skylab/LoomUtils.wdl (hunk 2 above)
Comment (kevinpalis) [reply tasks/skylab/LoomUtils.wdl:172]: Hi Kaylee. I made the changes. Thanks!

Comment (kayleemathews) [reply tasks/skylab/LoomUtils.wdl:172]: Thank you!

**Comment by kayleemathews** [comment]
@kevinpalis Thanks for making those doc updates!

**Pull request closed without merging**